hack/install_lxcfs_on_centos.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#
+# Install lxcfs from source code in CentOS
+#
+
+yes | yum install autotools-dev m4 autoconf2.13 autobook autoconf-archive gnu-standards autoconf-doc libtool
+yes | yum install fuse-devel.$(uname -p)
+yes | yum install pam-devel.$(uname -p)
+
+TMP=$(mktemp -d)
+trap "rm -rf $TMP" EXIT
+
+cd $TMP &&
+git clone https://github.com/lxc/lxcfs.git &&
```

Comment:
Should checkout to a stable branch  or commit_id

Reply:
I agree with what @zzchun said.